rfcs/rfc1/README.md

```diff
@@ -0,0 +1,211 @@
+
+* **Enhanced decision-making**: The RFC process provides a clear framework for evaluating proposals based on their merits, alignment with project goals, and potential impact. This structured approach helps the TSC make well-informed decisions that benefit the project as a whole.
+
+* **Clear accountability**: With the assignment of a Directly Responsible Individual (DRI) for each accepted RFC, there is clear ownership and accountability for the implementation of approved changes. This ensures that accepted proposals are properly executed and followed through.
+
+* **Maintained agility**: While the RFC process introduces some structure and formality, it is designed to maintain agility in the development process. The process allows for quick iteration and refinement of proposals through collaborative discussions and feedback.
+
+While the current informal approach may be more familiar and require less upfront effort, it lacks the benefits of a structured process in terms of transparency, collaboration, decision-making, and accountability. The RFC process strikes a balance between providing a clear framework for proposing and evaluating changes while still allowing for flexibility and agility in the development lifecycle.
+
+### <ins>Specify different types of RFCs</ins>
+
+As an alternative, we could adopt a categorization similar to [Python's PEP Types](https://legacy.python.org/dev/peps/pep-0001/#pep-types), which specifies three kinds of RFCs:
+
+1. A **Standards Track** RFC describes a new feature or implementation for the specifications or SDKs maintained by TBD. It may also describe an interoperability standard that will be supported outside the core libraries for current versions before a subsequent RFC adds core library support in a future version.
+
+2. An **Informational** RFC describes a design issue or provides general guidelines or information to the TBD community but does not propose a new feature.
+
+3. A **Process** RFC describes a process surrounding the development and maintenance of specifications and SDKs, or proposes a change to (or an event in) a process. Process RFCs are like Standards Track RFCs but apply to areas other than the specifications or SDKs themselves. They may propose an implementation, but not to the codebase. Examples include procedures, guidelines, and changes to the decision-making process.
+
+By adopting this categorization, the RFC process can differentiate between proposals that introduce new features or implementations (Standards Track), those that provide general information or guidelines (Informational), and those that propose changes to processes or procedures (Process). This distinction can help set expectations and guide the review and decision-making process for each type of RFC.
+
+## Prior art
```

Comment:
@blackgirlbytes also shared for consideration:
[Astro](https://github.com/withastro/roadmap?tab=readme-ov-file)
[Remix](https://remix.run/blog/open-development)
[A guide on approaching RFCs as a engineering org](https://leaddev.com/technical-decision-making/thorough-team-guide-rfcs)
[another guide on RFC approach](https://github.com/jakobo/rfc)

Reply:
thanks, kirah!